### PR TITLE
Bump XDR once more

### PR DESCRIPTION
**Pull request by 2opremio**
Bump XDR to [7015193879e9da7cb70d889a49eecf494fefe7e1](https://github.com/stellar/stellar-xdr/tree/7015193879e9da7cb70d889a49eecf494fefe7e1)

The integration tests won't work until soroban-rpc has been updated (the corresponding soroban-rpc PR is https://github.com/stellar/soroban-tools/pull/884 ).

### CODE REVIEW

File: services/horizon/docker/stellar-core-integration-tests.cfg [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING=true
+TESTING_SOROBAN_HIGH_LIMIT_OVERRIDE=true
```
Comment (2opremio) [inline services/horizon/docker/stellar-core-integration-tests.cfg:2]: This is going to break the protocol 19 integration tests. @tsachiherman could you please take a look and fix it like you did for captive core?

Comment (tsachiherman) [reply services/horizon/docker/stellar-core-integration-tests.cfg:2]: Sure! Will do.

Comment (tsachiherman) [reply services/horizon/docker/stellar-core-integration-tests.cfg]: I'm going to resolve this, as my change addresses that. let me know if you think it should be implemented differently.

**Comment by tsachiherman** [comment]
currently fails due to 
```
2023-08-30T15:12:51.3784642Z     sac_test.go:1072: 
2023-08-30T15:12:51.3785849Z         	Error Trace:	/home/runner/work/go/go/services/horizon/internal/integration/sac_test.go:1072
2023-08-30T15:12:51.3787270Z         	            				/home/runner/work/go/go/services/horizon/internal/integration/sac_test.go:50
2023-08-30T15:12:51.3787829Z         	Error:      	Received unexpected error:
2023-08-30T15:12:51.3797683Z         	            	horizon error: "Transaction Failed" (tx_failed, resource_limit_exceeded) - check horizon.Error.Problem for more information
2023-08-30T15:12:51.3798395Z         	Test:       	TestContractMintToAccount
```

I'm going to beef up logging around that one, but the core issue is likly to be solved in the rpc fee calculations.

File: services/horizon/docker/stellar-core-integration-tests.cfg (hunk 1 above)
Comment (2opremio) [reply services/horizon/docker/stellar-core-integration-tests.cfg]: Wait, I am confused. The problem wasn't being caused by the captive core file but from the core file.

Also, I don't think we should be enabling the high limits by default!

**Comment by 2opremio** [comment]
> I'm going to beef up logging around that one, but the core issue is likly to be solved in the rpc fee calculations.

Yes, I am on it with @sisuresh on Slack.

File: services/horizon/docker/stellar-core-integration-tests.cfg (hunk 1 above)
Comment (2opremio) [reply services/horizon/docker/stellar-core-integration-tests.cfg:2]: I ended up reverting the changes and adding a sub-protocol-20 configuration file

**Pull request merged**